wazo_confd/plugins/call_filter/resource.py

```diff
@@ -14,13 +15,30 @@
 class CallFilterList(ListResource):
     model = CallFilter
     schema = CallFilterSchema
+    call_filter_name_fmt = 'callfilter-{tenant_slug}-{callfilter_uuid}'
+
+    def __init__(self, tenant_dao, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self._tenant_dao = tenant_dao
 
     def build_headers(self, call_filter):
         return {'Location': url_for('callfilters', id=call_filter.id, _external=True)}
 
     @required_acl('confd.callfilters.create')
     def post(self):
-        return super().post()
+        form = self.schema().load(request.get_json())
+        form = self.add_tenant_to_form(form)
+
+        tenant = self._tenant_dao.get(form['tenant_uuid'])
+        # NOTE(afournier): we use a UUID as if it was the callfilter UUID but it's not
+        # Call filters do not use UUIDs yet
```

Comment:
Would it be hard to add the UUID column right now? This would avoid a painful migration (i.e. parsing names to get the UUID) later on when we do add the UUID column.

Reply:
It's another ticket completely. Call filters are far from being the only resource still using an integer ID.
It's also hard, requiring changes in function keys, fallbacks, etc.

Reply:
And how about adding the column `uuid` but keeping the `id` as primary key, only ensuring that the `uuid` is unique? That way we have the `uuid` in the database, we can use it for the callfilter name and the primary key migration `id` -> `uuid` will be easier.